neurom/apps/morph_stats.py

```diff
@@ -73,19 +82,59 @@ def _stat_name(feat_name, stat_mode):
     return '%s_%s' % (stat_mode, feat_name)
 
 
-def extract_stats(neurons, config):
+def _fill_compoundified(data, stat_name, stat):
```

Comment:
I think this needs a better name and description; also, does this work for all stats?

Reply:
The diff UI is confusing but I haven't changed this code. It was a nested function that I have simply extracted out so it should work as before. I can change its name and description though.